changelog/740.feature.rst

```diff
@@ -0,0 +1,8 @@
+Return the :class:`disnake.ui.ActionRow` instance on multiple methods to allow for fluent-style chaining.
+- :func:`ActionRow.append_item <disnake.ui.ActionRow.append_item>`
+- :func:`ActionRow.insert_item <disnake.ui.ActionRow.insert_item>`
+- :func:`ActionRow.add_button <disnake.ui.ActionRow.add_button>`
+- :func:`ActionRow.add_select <disnake.ui.ActionRow.add_select>`
+- :func:`ActionRow.add_text_input <disnake.ui.ActionRow.add_text_input>`
+- :func:`ActionRow.clear_items <disnake.ui.ActionRow.clear_items>`
+- :func:`ActionRow.remove_item <disnake.ui.ActionRow.remove_item>`
```

Comment:
```suggestion
- This applies to :func:`ActionRow.append_item <disnake.ui.ActionRow.append_item>`, :func:`ActionRow.insert_item <disnake.ui.ActionRow.insert_item>`, :func:`ActionRow.add_button <disnake.ui.ActionRow.add_button>`, :func:`ActionRow.add_select <disnake.ui.ActionRow.add_select>`, :func:`ActionRow.add_text_input <disnake.ui.ActionRow.add_text_input>`, :func:`ActionRow.clear_items <disnake.ui.ActionRow.clear_items>`, and :func:`ActionRow.remove_item <disnake.ui.ActionRow.remove_item>`.
```
takes up a lot of space otherwise, could probably even remove the `ActionRow.` prefix since it can be inferred from the context

Reply:
I kinda like how it is now.

Reply:
NVM, I changed it locally and suppose this is better. f949e65a